src/Markdown.js

```diff
@@ -56,23 +57,47 @@ export default class Markdown {
         return is_plain;
     }
 
-    toHTML() {
+    render(html) {
```

Comment:
Given most of this function is branching based on this conditional, separate functions sounds excellent, as it will also make it clearer at the call site what you're actually doing `toHTML` or `toPlaintext` or whatever.